src/corecel/Assert.cc

```diff
@@ -138,6 +138,8 @@ const char* to_cstring(RuntimeErrorType which)
             return "mpi";
         case RuntimeErrorType::geant:
             return "geant4";
+        case RuntimeErrorType::root:
+            return "ROOT";
```

Comment:
I think this is just for output to the JSON error wrapper, but we should be consistent about capitalization... would you lowercase `root` or add capitalization to Geant4, MPI, CUDA?

Reply:
For `ROOT` the case helps (a little) to reduce the confusion with the `root` unix username.  So I will add casing to the others.